apps/app/project.json

```diff
@@ -62,6 +62,13 @@
         "command": "env-cmd -f .env.boson npx playwright test"
       }
     },
+    "e2e:ui": {
```

Comment:
in some ways, it might be easier to run this test on elements to simulate all the various org settings that could occur... like disabling on enabling combine, orders, etc.